codec.proto

```diff
@@ -231,10 +226,11 @@ message Timestamp {
 }
 
 message TxResult {
-  uint64            height = 1;
-  uint32            index  = 2;
-  Tx                tx     = 3;
-  ResponseDeliverTx result = 4;
+  bytes             hash   = 1;
```

Comment:
As this is an additional field we added, I would put this as the last field (`= 5`). That way if we receive a native TxResult, it should still decode properly.

Reply:
okay, good idea